presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -866,6 +866,36 @@ Examples:
 
     CALL iceberg.system.remove_orphan_files(schema => 'db', table_name => 'sample');
 
+Fast Forward Branch
+^^^^^^^^^^^^^^^^^^^
+
+This procedure advances the current branch to a more recent snapshot from another branch without replaying any intermediate snapshots.
```

Comment:
Should it be `current snapshot of the specified branch` rather than `current branch`? As I understand, the specified branch to fast-forward may not necessarily be the current branch.